Sts1CobcSw/Periphery/Rf.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <Sts1CobcSw/Periphery/RfNames.hpp>
+#include <Sts1CobcSw/Serial/Byte.hpp>
+
+#include <array>
+#include <cstdint>
+#include <span>
+#include <string_view>
+
+
+namespace sts1cobcsw::periphery::rf
+{
+auto Initialize() -> void;
+auto InitializeGpioAndSpi() -> void;
```

Comment:
This is should not be a public function → move the declaration to the source file.